Sources/Xcodeproj/pbxproj().swift

```diff
@@ -272,7 +272,8 @@ func xcodeProject(
     // source directory of each target to the corresponding group, so that
     // source files added later will be able to find the right group.
     @discardableResult
-    func createSourceGroup(named groupName: String, for targets: [ResolvedTarget], in parentGroup: Xcode.Group) -> Xcode.Group? {
+    func createSourceGroup(named groupName: String, for resolvedTargets: [ResolvedTarget], in parentGroup: Xcode.Group) -> Xcode.Group? {
+        let targets = resolvedTargets.sorted { (left, right) -> Bool in left.name < right.name }
```

Comment:
You can just do: `let targets = targets.sorted({ $0.name < $0.name })`

Reply:
👍